prometheus-exporters-formula/metadata/form.yml

```diff
@@ -65,3 +65,55 @@ postgres_exporter:
     $name: "Arguments"
     $type: text
     $help: Please refer to the documentation for available options.
+
+# SAP/HA exporters
+ha_cluster_exporter:
+  $type: group
+  $help: Prometheus exporter for Pacemaker based HA clusters.
+
+  enabled:
+    $type: boolean
+    $default: False
+
+  address:
```

Comment:
This address should be passed to the exporter as an argument.

Reply:
just as understaing question why we don't do it for the other exporters?